### DISCUSSION TITLE
Trying to set spell check language

### DISCUSSION THREAD

**Discussion by B1azA** [LunarVim/LunarVim#3634] [Q&A] [answered]
Hi, I am trying to set spell check language to czech because I need it for taking notes. 
I am using this command:
:setlocal spell spelllang=cs
It works in Neovim and Vim but not in LunarVim.

It shows this:

> spellfile#LoadFile(): No (writable) spell directory found.

> Failed to create: /home/b1aza/.local/share/nvim/site/spell

> Warning: Cannot find word list "cs.utf-8.spl" or "cs.ascii.spl"

Does anybody have a solution?

**Comment by LostInTheLogs** [comment, marked as answer]
Create a spell directory in .config/lvim
